buildah/buildah.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: build.knative.dev/v1alpha1
+kind: BuildTemplate
+metadata:
+  name: buildah
+spec:
+  parameters:
+  - name: BUILDER_IMAGE
+    description: The location of the buildah builder image.
+  - name: IMAGE
+    description: The name of the image to push.
+  - name: DOCKERFILE
+    description: Path to the Dockerfile to build.
+    default: ./Dockerfile
+
+  steps:
+  - name: build
+    image: ${BUILDER_IMAGE}
+    args: ['--storage-driver=vfs', 'bud', '-f', '${DOCKERFILE}', '-t', 'image', '.']
```

Comment:
Why you need vfs?
As long as /var/lib/containers is mounted as an emptydir volume, overlay shoud work?
https://github.com/containerbuilding/cbi/blob/c9c26dc7236e99e8a565ee8ebd33bef75352f578/pkg/plugin/backends/buildah/buildah.go#L87-L107

Reply:
@AkihiroSuda true enough. if that mountpoint is xfs or ext4. It was not working for me though. And for now, vfs does work. We can circle back on that, because I would much prefer to use overlayfs (and to not use vfs) driver.

Reply:
Which kernel version is this running on?  Is the overlay driver loaded (i.e., is it listed in /proc/filesystems)?

Reply:
@AkihiroSuda i wondered that too, but `findmnt /var/lib/containers` inside the container shows
```
/var/lib/containers /dev/sda2[/var/lib/kubelet/pods/f58f320f-999c-11e8-a436-1c1b0d0ece5a/volumes/kubernetes.io~empty-dir/varlibcontainers] xfs    rw,relatime,attr2,inode64,noquota
```